introduction.md

```diff
@@ -1,3 +1,7 @@
 Watashi wa o-namae wa Gabin deska!
 My Fav Codeschool path is Git, yeah!
 or Ruby
+*Git
+*Ruby
+*VMWARE
+*Networks
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?